doc/nrf/protocols.rst

```diff
@@ -20,5 +20,6 @@ They introduce you to concepts that are important to work with the protocol and
    protocols/thread/index
    protocols/zigbee/index
    protocols/wifi/index
+   protocols/coexistence/index
 
 For details on Wi-Fi coexistence with short range protocols (like Bluetooth LE or Thread), see the :ref:`ug_radio_coex` page.
```

Comment:
Can this line be moved to the new page? One, it mentions coexistence. Two, we should not have any content after the list of subpages.

Reply:
As per the offline discussion, deleted this line with the link to the https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_dev/wifi_coex/index.html#ug-radio-coex page, as the matrix in the new Wireless coexistence page includes this link.